README.md

```diff
@@ -12,7 +11,6 @@
   <img  height=230px src="https://user-images.githubusercontent.com/6642037/124296829-afecbf80-db5a-11eb-9dbc-d92b588e1f42.png" alt="Project Homepage"></a>
 </p>
 
-
```

Comment:
Hi.

This change to readme has to be reverted. 
git checkout develop readme.md

Reply:
Hi Niels, will revert it .